sweetest/src/main/java/com/mysugr/sweetest/framework/dependency/DependencySetup.kt

```diff
@@ -28,28 +28,24 @@ object DependencySetup {
     inline fun <reified T : Any> addRealRequired(
         alias: KClass<*>? = null,
         noinline realInitializer: DependencyInitializer<T>
-    ):
-        DependencyConfiguration<T> =
-        setupHandler.addConfiguration(
-            T::class, realInitializer, null,
-            DependencyMode.REAL, alias)
+    ): DependencyConfiguration<T> =
```

Comment:
i wonder about this auto-format. was the column count not set correctly? 🤔 
did you do it with ktlint's auto-formatter? maybe it doesn't pick up the correct max line length...